src/lib/Behat/BusinessContext/ContentUpdateContext.php

```diff
@@ -28,6 +29,20 @@ public function iSetFields(TableNode $table): void
         }
     }
 
+    /**
+     * @When I set content fields for user
+     */
+    public function iSetFieldsForUser(TableNode $table): void
+    {
+        $updateItemPage = PageObjectFactory::createPage($this->utilityContext, UserCreationPage::PAGE_NAME, '');
+        $updateItemPage->verifyIsLoaded();
+        $hash = $table->getHash();
+        foreach ($hash as $row) {
```

Comment:
```suggestion
        foreach ($table->getHash() as $row) {
```
Nitpick: IMHO the $hash variable is not needed and it's more readable when it's omitted.